content/chat/moderation/index.textile

```diff
@@ -30,3 +34,11 @@ h3(#after-publish). After publish
 When using after publish moderation, a message is published as normal, but is forwarded to a moderation engine after the fact. This allows you to avoid the latency penalty of vetting content prior to publish, at the expense of bad content being visible in the Chat room (at least briefly). Many automated moderation solutions are able to process and delete offending messages within a few seconds of publication.
 
 Please note that message deletion is currently performed as a soft delete, meaning that your application will need to filter out deleted messages that it sees.
+
+h2(#directvsbyo). Direct vs Bring Your Own
+
+There are a plethora of moderation options available on the market, from simple pattern-matching APIs to fully fledged machine learning models.
+
+Ably provides direct integrations between your chat room and various moderation providers, to give you access to powerful moderation platforms with minimal code and setup. If a provider you are looking for is not listed, please get in touch!
+
+Alternatively, you might have a custom solution you wish to integrate with, or a provider that Ably doesn't yet directly support. If this is the case, Ably offers a "Bring Your Own" option, where you can utilise serverless functions such as AWS Lambda or other transports to reach out to your own infrastructure to moderate chat messages. Again, if a transport you're looking for isn't available, please get in touch.
```

Comment:
>or other transports

This also doesn't seem true right now, no?

Reply:
Changed wording